src/user.rs

```diff
@@ -266,6 +270,11 @@ impl User {
         self.client.read_messages(room_id).await;
     }
 
+    async fn get_channel_members(&self, room_id: OwnedRoomId) {
+        log::debug!("user '{}' act => {}", self.localpart, "GET CHANNEL MEMBERS");
+        self.client.get_channel_members(&room_id).await
```

Comment:
should we do something with the result?

Reply:
IDK. nothing specified I am not sure if it's useful to store it or to do something like that. @moliva what do u think?